modules/standards/package.json

```diff
@@ -4,42 +4,29 @@
   "description": "",
   "repository": "phenyl-js/phenyl",
   "license": "Apache License 2.0",
-  "author": "Shin Suzuki <shinout310@gmail.com> ,Nicholas Lee <nicholaslee119@gmail.com>",
+  "author": "Shin Suzuki <shinout310@gmail.com>",
   "files": [
-    "src",
-    "dist",
-    "package.json",
-    "jsnext.js",
+    "lib",
     "LICENSE",
-    "README.md",
-    "decls"
+    "README.md"
   ],
-  "main": "lib/index.js",
-  "types": "lib/index.d.js",
+  "main": "lib/index.ts",
```

Comment:
main file should still be `lib/index.js` because the file is compiled into `.js`